docs/_templates/sse.html

```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
```

Comment:
is it really a documentation template? maybe you should move it to `examples/sse/templates/index.html`